R/download_link.R

```diff
@@ -0,0 +1,130 @@
+#' Download link
+#'
+#' @param outputId The name of the output slot that the `downloadHandler` is assigned
+#' to.
+#' @param link_text Text that will appear describing the download action
+#' (default: "Download file").
+#' Vague text like 'click here' or 'here' will cause an error, as will ending in
+#' a full stop. Leading and trailing white space will be automatically trimmed.
+#' If the string is shorter than 7 characters a console warning will be thrown.
+#' There is no way to hush this other than providing more detail.
+#' @param file_type The file type to be download (default: CSV)
+#' @param file_size The file size if known. Needs to be a string ending in one
+#' of KB, MB, GB or rows.
+#'
+#' @returns shiny.tag object
+#' @export
+#'
+#' @examples
```

Comment:
I think it would be worth including an example showing the file size (either in the example you have, or as an additional one)